atom/browser/lib/init.coffee

```diff
@@ -9,6 +9,27 @@ process.argv.splice 1, 1
 
 # Add browser/api/lib to require's search paths,
 # which contains javascript part of Atom's built-in libraries.
+if process.platform is 'win32'
+  homeDir = process.env.USERPROFILE
+else
+  homeDir = process.env.HOME
+
+syspath = []
+
+if homeDir
+  syspath.push(path.resolve(homeDir, '.node_modules'))
```

Comment:
We can simply remove the first 2 elements of `module.globalPaths` instead of detecting duplicated paths, and we should also add a comment of why we did it.